lib/livebook_web/live/output.ex

```diff
@@ -346,8 +346,11 @@ defmodule LivebookWeb.Output do
     """
   end
 
-  defp render_output(%{type: :error, message: message}, %{}) do
-    render_formatted_error_message(message)
+  defp render_output(%{type: :error, message: message}, %{id: id}) do
+    assigns = %{id: id, message: message}
+    ~H"""
+    <.live_component module={Output.TerminalTextComponent} id={@id} text={@message} />
```

Comment:
I think it would be better to replicate the copy button in `render_formatted_error_message` directly. The `TerminalTextComponent` does line virtualization, which messes with manual text selection sometimes, while errors are rather short and static, so we don't need that :)

Reply:
done..